plugins/modules/s3_bucket.py

```diff
@@ -133,6 +133,12 @@
     choices: [ 'BucketOwnerEnforced', 'BucketOwnerPreferred', 'ObjectWriter' ]
     type: str
     version_added: 2.0.0
+  object_lock_enabled:
+    description:
+      - Whether S3 Object Lock to be enabled.
+      - Defaults to C(False) when creating a new bucket.
+    type: bool
+    version_added: 5.3.0
```

Comment:
Changed the version_added, we plan on backporting this to 5.3.0

Moved this up a little higher so it's away from the `dualstack` option which isn't being backported (it's related to more complex changes)